examples/train.py

```diff
@@ -9,7 +9,7 @@
 from stable_baselines3.common.utils import set_random_seed, get_schedule_fn
 from stable_baselines3.common.vec_env import SubprocVecEnv
 
-from agent_policy import AgentPolicy
+from .agent_policy import AgentPolicy
```

Comment:
This seems to throw an error like:

```
(kaggle_lux2) D:\Repos\LuxPythonEnvGym>python ./examples/train.py
Traceback (most recent call last):
  File "./examples/train.py", line 12, in <module>
    from .agent_policy import AgentPolicy
ImportError: attempted relative import with no known parent package
```

I think agent_policy library is not part of the package, so should be referenced like `from agent_policy import AgentPolicy`, right?

Reply:
Right!

Reply:
Ok, I just put in a quick PR #82 and merged to change two minor things so the example works.

Reply:
Without the dot it does not import for me in jupyter notebook, But it is ok, I will change for myself locally. And good idea to remove the prints, too much log pollution from them.